moveit_core/kinematic_constraints/src/kinematic_constraint.cpp

```diff
@@ -597,7 +597,8 @@ bool OrientationConstraint::configure(const moveit_msgs::msg::OrientationConstra
   // clearing out any old data
   clear();
 
-  link_model_ = robot_model_->getLinkModel(oc.link_name);
+  bool found;
```

Comment:
What is this variable needed for? Would it make sense to remove it and if no, do you mind initializing it with `false`

Reply:
This is just needed to silent an additional error message in `getLinkModel()`:
https://github.com/moveit/moveit2/blob/d5f763ed18b4a0cedcb2efc40961885f30387146/moveit_core/robot_model/src/robot_model.cpp#L1348-L1363

The variable is initialized there.

Reply:
Sure.